src/deprecated.jl

```diff
@@ -1,3 +1,4 @@
+import Base: isidentifier, is_id_start_char, is_id_char
```

Comment:
I hadn't noticed this. Better fully qualify these, so that we can remove the deprecations separately without having unrelated code at the top of the file.

Reply:
fixed